ios/FluentUI.Demo/FluentUI.Demo/SwiftUI/FluentUIDemoToggle.swift

```diff
@@ -11,7 +11,7 @@ struct FluentUIDemoToggle: View {
     var isOn: Binding<Bool>
 
     let switchToggleStyle: SwitchToggleStyle = {
-        return SwitchToggleStyle(tint: Color(Colors.communicationBlue))
+        return SwitchToggleStyle(tint: Color(UIColor(colorValue: GlobalTokens.brandColors(.comm80))))
```

Comment:
little orthogonal but we still have  reference to brandingColor in global tokens? also, i think the toggle color should just be the basic branding background color

Reply:
Good catches!

Reply:
In this circumstance, we'd have to use the shared theme. Would you prefer the shared theme with branding BG, or comm80?

Reply:
It would be too costly to fully enable theming for this control in this PR. If we want to update it in the future, that's a good idea, but for now I will leave it for another change.